qbft/messages.go

```diff
@@ -108,6 +75,12 @@ func (msg *Message) Decode(data []byte) error {
 	return msg.UnmarshalSSZ(data)
 }
 
+func GetMessageFromBytes(data []byte) (*Message, error) {
```

Comment:
suggest to rename to `DecodeMessage` move right below the struct definition (just to conform to Go standards)

Reply:
Yes, better. Thanks!